### PR TITLE
✨Source Drift: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #28113: Drift

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Drift to Low-Code CDK*
Closes #28113

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

image

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Drift</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by btkcodedev** [comment]
Full test results: Passed state
image

**Comment by btkcodedev** [comment]
Ping via slack for credentials
ID: btkcodedev

File: airbyte-integrations/connectors/source-drift/acceptance-test-config.yml [hunk 1]
```diff
@@ -1,20 +1,38 @@
...
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      empty_streams: ["accounts", ]
+    tests:
+      - config_path: "secrets/config.json"
+        configured_catalog_path: "integration_tests/configured_catalog.json"
+        empty_streams:
+          - name: users
+            bypass_reason: "Sandbox account can't seed this stream"
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-drift/acceptance-test-config.yml:21]: Need to add `accounts` into the empty streams.

File: airbyte-integrations/connectors/source-drift/metadata.yaml [hunk 2]
```diff
@@ -1,27 +1,25 @@
...
   githubIssueLabel: source-drift
   icon: drift.svg
   license: MIT
   name: Drift
-  registries:
-    cloud:
-      enabled: false # hide Source Drift https://github.com/airbytehq/airbyte/issues/24270
-    oss:
-      enabled: true
+  releaseDate: TODO
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-drift/metadata.yaml:19]: This needs a valid date in `YYYY-MM-DD` format. (today's date is fine.

**Review by archangelic — CHANGES_REQUESTED** [review]
Thanks for your contribution! This looks good, but I have listed out some small changes below.

File: airbyte-integrations/connectors/source-drift/source_drift/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    authenticator:
+      type: BearerAuthenticator
+      api_token: "{{ config['credentials']['access_token'] }}"
+
+  retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+    paginator:
+      type: NoPagination
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-drift/source_drift/manifest.yaml:23]: This needs to have a paginator. For example:
```
base_paginator:
    type: "DefaultPaginator"
    pagination_strategy:
      type: "CursorPagination"
      cursor_value: "{{ last_records['nextPageToken'] }}"
    page_token_option:
      type: "RequestPath"
      field_name: "pageToken"
      inject_into: "request_parameter"
```

File: airbyte-integrations/connectors/source-drift/source_drift/manifest.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+    $ref: "#/definitions/base_stream"
+    name: "users"
+    primary_key: "id"
+    $parameters:
+      path: "/users"
+
+streams:
+  - "#/definitions/accounts_stream"
+  - "#/definitions/conversations_stream"
+  - "#/definitions/users_stream"
```
Comment (archangelic) [inline airbyte-integrations/connectors/source-drift/source_drift/manifest.yaml:64]: `contacts` and `messages` streams need to be migrated as well.

**Review by archangelic — CHANGES_REQUESTED** [review]

**Comment by btkcodedev** [comment]
@archangelic All comments resolved
Test result- passed state :)
image

**Review by archangelic — APPROVED** [review]

**Pull request merged**